### PR TITLE
Removing target dependency to SetBadgeProps on publish targets.

### PR DESCRIPTION
**Pull request by livarcocc**
Removing target dependency to SetBadgeProps on publish targets. That's not needed on the repo-refactored CLI.

### CODE REVIEW

**Comment by livarcocc** [comment]
@dotnet-bot Test RHEL7.2 x64 Release Build

**Comment by livarcocc** [comment]
@dotnet-bot Test Ubuntu x64 Release Build

File: run-build.ps1 [hunk 1]
```diff
@@ -106,13 +106,13 @@ if($Architecture.StartsWith("arm", [StringComparison]::OrdinalIgnoreCase))
 Write-Output "$dotnetInstallPath -version ""$dotnetCliVersion"" -InstallDir $env:DOTNET_INSTALL_DIR -Architecture ""$InstallArchitecture"""
 Invoke-Expression "$dotnetInstallPath -version ""$dotnetCliVersion"" -InstallDir $env:DOTNET_INSTALL_DIR -Architecture ""$InstallArchitecture"""
 
-Invoke-Expression "$dotnetInstallPath -Version ""1.1.2"" -SharedRuntime -InstallDir $env:DOTNET_INSTALL_DIR -Architecture ""$Architecture"""
-
 if ($LastExitCode -ne 0)
```
Comment (peterhuene) [inline run-build.ps1]: Should this check now be removed or were we missing corresponding checks for the previous `Invoke-Expression`?

Comment (livarcocc) [reply run-build.ps1]: I still want the check to happen for the regular SDK. I just don't want it to fail when installing the backwards compatible runtime, as it won't be available on all OSes.

Comment (peterhuene) [reply run-build.ps1]: Cool, just wasn't sure if we still needed it since no check was done for the preceding command before.

**Pull request merged**